server/middlewares/strategies/gitlab.js

```diff
@@ -0,0 +1,34 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const OAuth2Strategy = require('passport-oauth').OAuth2Strategy;
+const breq = require('bluereq')
+const User = __.require('controllers', '/users/lib/users')
+
+const GitlabOAuth2Strategy = new OAuth2Strategy({
+  authorizationURL: CONFIG.get('auth.gitlab.authorizationURL'),
+  tokenURL: CONFIG.get('auth.gitlab.tokenURL'),
+  clientID: CONFIG.get('auth.gitlab.clientID'),
+  clientSecret: CONFIG.get('auth.gitlab.clientSecret'),
+  callbackURL: CONFIG.get('auth.gitlab.callbackURL'),
+}, function(accessToken, refreshToken, profile, done) {
+  // smell: `profile` should be filled with user info, instead requesting user info through accessToken endpoint
```

Comment:
What does this mean? Is it even worth an issue after merging?

Reply:
I detailed the comment in the new commits https://github.com/jum-s/data.transformap.co/blob/425-oauth2-provider/server/middlewares/strategies/gitlab.js#L14